lms/djangoapps/verify_student/models.py

```diff
@@ -1251,3 +1256,13 @@ def retire_user(cls, user_id):
         """
         verification_attempts = cls.objects.filter(user_id=user_id)
         verification_attempts.delete()
+
+    hide_status_from_user = models.BooleanField(
+        default=False,
+        null=True,
+    )
```

Comment:
That makes sense to me, and I think that's a sensible default!

Your `create_verification_attempt` function has a default of `None` for `hide_status_from_user`. Won't that set this value to `null` by default? If you want this value to be set to `False` by default, then I think the default in the API function should be `False` instead.

On the other hand, is the purpose of the option of setting `null` on this field to indicate a third state beyond `True` and `False` - something like "this IDV implementation didn't have an opinion on the value of this field"? Because from the perspective of `should_display_status_to_user`, which only returns `True` or `False`, `null` doesn't have a purpose, but I could see why you might want it - just trying to better understand the rationale!

Reply:
Also, just curious about the decision to model this as the inverse of `should_display_status_from_user` (i.e. "should hide" instead of "should show"). Is that because we assume that, most of the time, we'd want to show, and the exception should be to hide?

Reply:
To answer your questions:
- The default value of `hide_status_from_user` should be `False`, which I'm changing now.
- We are assuming that we'd usually want to show status, with the exception being to hide it.